media.css

```diff
@@ -0,0 +1,288 @@
+@media screen and (min-width: 420px) and (max-width: 912px) {
+  body {
+    max-width: 650px;
+    margin: 0 auto;
+  }
+
+  .img {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 400px;
+  }
+
+  .useds {
+    width: 90%;
+    justify-content: space-between;
+    margin: 37px auto;
+  }
+
+  .used {
+    width: 10%;
+  }
+
+  .contact h3 {
+    font-size: 32px;
+    color: #9390ff;
+    margin-top: 120px;
+  }
+
+  .contact .text {
+    margin: 50px auto;
+    width: 320px;
+    text-align: center;
+  }
+}
+
+@media screen and (min-width: 900px) and (min-width: 1100px) {
```

Comment:
- [ ] You have possibly erroneously specified `min-width` twice. Please ensure that your desktop `min-width` is between `992px `and `1100px` 👍